.releaserc

```diff
@@ -0,0 +1,59 @@
+{
+	"branches": ["master", "pre-release"],
```

Comment:
You should mark the new branch as pre-release:
https://semantic-release.gitbook.io/semantic-release/recipes/release-workflow/pre-releases

You'll also want to add that branch name to the on push hook in the CI yaml file.